build.gradle

```diff
@@ -105,7 +105,7 @@ allprojects {
     java {
       // This path needs to be relative to each project
       target fileTree('.') {
-        include '**/*.java'
+        include '**/src/*/java/**/*.java'
```

Comment:
what are these changes for??

Reply:
meant for another PR.  Spotless was catching intellij generated files from .../out/classes/....  By limiting src to having to live in src directories generated and transient java files don't get style checked.